docs/MULTIMODELSERVING_GUIDE.md

```diff
@@ -0,0 +1,44 @@
+# Multi-Model Serving
+## Introduction
+
+### Problem
+
+With machine learning approaches becoming more widely adopted in organizations, there is a trend to deploy many models. More models aims to provide personalized experience which often need to train a lot of models. Additionally, many models help to isolate each user’s data and train models separately for data privacy.
+Unfortunately, KFserving follows a one model and one server paradigm which presents a challenge for the Kubernetes cluster.
```

Comment:
When KFServing was originally designed, it follows the one model one server ....